README.md

```diff
@@ -138,6 +147,25 @@ archive.append('hello.txt', function () {
 })
 ```
 
+#### `archive.checkpoint(version, [callback])`
+
+Write a version checkpoint to the archive, marking the state for future reference. Version includes:
+
+```js
+{
+  label: '1.0.0', // can be any string, but semantic versions are recommended
+  message: 'The first version!' // an optional string describing the checkpoint
+}
+```
+
+If the label has been used before, Hyperdrive will error.
```

Comment:
to me `label` isn't a term for an unique datum, maybe let's use `version` instead?